src/backtrace/mod.rs

```diff
@@ -66,6 +66,33 @@ pub unsafe fn trace_unsynchronized<F: FnMut(&Frame) -> bool>(mut cb: F) {
     trace_imp(&mut cb)
 }
 
+/// Similar to [trace_unsynchronized], but additionally, it supports tracing a thread other than the current thread.
+///
+/// It gets the traced thread's handle as its first argument.
+///
+/// # Safety
+///
+/// This function is intended for profiling and debugging.
+///
+/// If the given thread handle is not the current thread,
+/// this function suspends the thread for a short amount of time to be able to extract the thread context.
+/// This might end in a deadlock if the current thread tries to access an object thats been locked by the suspended thread.
```

Comment:
Are these statements about the briefness and the concern about deadlocking related to each other?